db-client-java/src/main/java/com/eventstore/dbclient/GrpcClient.java

```diff
@@ -276,7 +280,7 @@ private void messageLoop() {
     private void closeConnection() {
         if (this.channel != null) {
             try {
-                this.channel.shutdown().awaitTermination(Timeouts.DEFAULT.shutdownTimeout, Timeouts.DEFAULT.shutdownTimeoutUnit);
+                this.channel.shutdown().awaitTermination(5, TimeUnit.SECONDS);
```

Comment:
The same concern as above: https://github.com/EventStore/EventStoreDB-Client-Java/pull/142/files#r853930699.

Reply:
I would argue that deadlines are for operations only and nothing in the RFC mentions anything for connection management. The Java gRPC lib is the only one with an async close function. I set five because it was the previous default value for shutdowns.

Reply:
The RFC, in my opinion, states the clear deadlines as 10 seconds. @hayley-jean, thoughts?

Reply:
The deadline only applies to the operations though, this looks like it's a timeout for closing the gRPC connection which isn't covered by any of the RFCs.

If the previous default was 5 seconds, then I think it would be best not to change it